src/config/define.auto.c

```diff
@@ -2276,6 +2276,44 @@ static ConfigDefineOptionData configDefineOptionData[] = CFGDEFDATA_OPTION_LIST
         )
     )
 
+    // -----------------------------------------------------------------------------------------------------------------------------
+    CFGDEFDATA_OPTION
+    (
+        CFGDEFDATA_OPTION_NAME("pg-database")
+        CFGDEFDATA_OPTION_REQUIRED(true)
+        CFGDEFDATA_OPTION_SECTION(cfgDefSectionStanza)
+        CFGDEFDATA_OPTION_TYPE(cfgDefOptTypeString)
+        CFGDEFDATA_OPTION_INTERNAL(false)
+
+        CFGDEFDATA_OPTION_INDEX_TOTAL(8)
+        CFGDEFDATA_OPTION_SECURE(false)
+
+        CFGDEFDATA_OPTION_HELP_SECTION("stanza")
+        CFGDEFDATA_OPTION_HELP_SUMMARY("PostgreSQL database user.")
+        CFGDEFDATA_OPTION_HELP_DESCRIPTION
+        (
+            "The database name used when connecting to PostgreSQL. The default is usually best but some installations may not "
+                "contain this database.\n"
+            "\n"
+            "Note that for legacy reasons the setting of PGDATABASE will be ignored."
```

Comment:
this looks weird - isn't the option pg-database and not PGDATABASE?

Reply:
PGDATABASE is a well-known env variable. Added a clarification.